source/19-Psalms.usfm.db

```diff
@@ -5772,7 +5772,7 @@
 \b
 \b
 \q3
-\v 52 \em Blest be the \+nd Lord\+nd*, for ever and ever.\em*
+\v 52 \em Blest be the \+nd Lord\+nd*, [us:forever|cth:for ever] and ever.\em*
```

Comment:
I think that the "for ever" here should either remain split given the fact that it is part of the phrase "for ever and ever" or that the entire "for ever and ever" be replaced with "forever".